src/main/kotlin/de/livepoll/api/service/PollService.kt

```diff
@@ -130,6 +131,25 @@ class PollService(
         }
     }
 
+    fun getNextPollItem(pollId: Long): PollDtoOut {
```

Comment:
Do I get the answers when you send me `PollDtoOut`. Maybe I miss something, but I think `PollDtoOut` does not contain any information about answers / answerCount?!

Reply:
Yes, you are right. I fixed it